addons/mass_mailing/models/mailing_list.py

```diff
@@ -11,6 +11,9 @@ class MassMailingList(models.Model):
     _order = 'name'
     _description = 'Mailing List'
 
+    # As this model has his own data merge, avoid to enable the generic data_merge on that model.
+    _disable_data_merge = True
```

Comment:
No need to have a void line before comment, we can have all class attributes listed in a row